tests/cdrouter_bootstrap.py

```diff
@@ -176,21 +186,22 @@ def add_cdrouter_config(config):
                     break
 
             # TODO: mask from config? wanNatIp vs. wanIspAssignGateway?
-            contents=contents + """
-testvar wanMode static
-testvar wanIspIp %s
-testvar wanIspGateway %s
-testvar wanIspMask 255.255.255.0
-testvar wanIspAssignIp %s
-testvar wanNatIp %s
-testvar IPv4HopCount %s
-testvar lanDnsServer %s
-testvar wanDnsServer %s""" % (self.config.board['cdrouter_wanispip'], \
-                              self.config.board['cdrouter_wanispgateway'], \
-                              wan_ip, wan_ip, \
-                              self.config.board['cdrouter_ipv4hopcount'], \
-                              board.get_dns_server(), \
-                              board.get_dns_server_upstream())
+            if self.config.board['cdrouter_location'] != 'CBN':
+                contents=contents + """
+    testvar wanMode static
+    testvar wanIspIp %s
+    testvar wanIspGateway %s
+    testvar wanIspMask 255.255.255.0
+    testvar wanIspAssignIp %s
+    testvar wanNatIp %s
+    testvar IPv4HopCount %s
+    testvar lanDnsServer %s
+    testvar wanDnsServer %s""" % (self.config.board['cdrouter_wanispip'], \
+                                  self.config.board['cdrouter_wanispgateway'], \
+                                  wan_ip, wan_ip, \
+                                  self.config.board['cdrouter_ipv4hopcount'], \
+                                  board.get_dns_server(), \
+                                  board.get_dns_server_upstream())
```

Comment:
These need to be defined in the boardfarm config for CBN... not just overridden. They should also be removed from configs/cdrouter-mv1.conf only generic items should live in that config.